src/components/stepforms/schemas/delete.ts

```diff
@@ -0,0 +1,26 @@
+const schema = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  title: 'Delete column step',
+  type: 'object',
+  properties: {
+    name: {
+      type: 'string',
+      enum: ['delete'],
+    },
+    column: {
+      type: 'string',
+      minLength: 1,
+      title: 'Column to delete',
+      description: 'Column to delete',
+      attrs: {
+        placeholder: 'Enter a column',
+      },
+    },
+  },
+  required: ['name', 'column'],
+  additionalProperties: false,
+};
+
+export default function buildSchema(_form: any) {
```

Comment:
Just to make I understand your point, you would prefer writing:

```javascript
export default { ... }
```

rather than

```javascript
export default() => { ... }
```

I'm ok with it, although it would probabliy require a `typeof ` test in the caller code.

Reply:
Not sure it is the same question than @Dalanir but: do we need to go through buildSchema here ? It does nothing here right ? So I think the question was: can't we just export it directly like the argmin schema for exemple ?

Reply:
That's exactly what I wanna say @vdestraitt